grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -82,4 +100,12 @@ class ClientJobStatus {
             get(explorer, id)
         }
     }
+
+    static List<ClientJobStatus> getAllForTransaction(@Nonnull final JobExplorer explorer, @Nonnull final long transactionID) {
+        if(!explorer) throw new IllegalArgumentException("jobExplorer == null")
```

Comment:
Minor: `explorer != null` is clearer. (And helps highlight the issue on the line below...)